src/offsetPath.js

```diff
@@ -2,6 +2,34 @@
 'use strict';
 
 (function () {
+  function basicShapePolygon (input) {
+    // TODO: Support the fill-rule option and %
```

Comment:
By the way, fill-rule will have no effect. If it is present, we will check it is valid, and store it so we can serialize, but it won't affect the transform.  (No action needed for this pull request.)